Makefile.dep

```diff
@@ -43,6 +43,12 @@ ifneq (,$(filter gnrc_zep,$(USEMODULE)))
   USEMODULE += vtimer
 endif
 
+ifneq (,$(filter gnrc_tftp_%,$(USEMODULE)))
+  USEMODULE += gnrc_udp
+  USEMODULE += gnrc_tftp
+  USEMODULE += vtimer
```

Comment:
If I remove it I get:

    /home/es_user/RIOT/examples/gnrc_tftp/bin/native/cpu.a(syscalls.o): In function `_gettimeofday':
    /home/es_user/RIOT/cpu/native/syscalls.c:405: undefined reference to `vtimer_gettimeofday'
    collect2: error: ld returned 1 exit status
    /home/es_user/RIOT/examples/gnrc_tftp/../../Makefile.include:213: recipe for target 'all' failed
    make: *** [all] Error 1